exercise2/app/src/main/java/com/example/codechallange2/MainActivity1.java

```diff
@@ -0,0 +1,39 @@
+package com.example.codechallange2;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.widget.ScrollView;
+
+public class MainActivity1 extends AppCompatActivity {
+    ScrollView sv;
```

Comment:
Please avoid having 2 letters names for fields. The names are not helping the reader of the code to understand their purpose